src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/ciagkpi/PefCiagKpiService.kt

```diff
@@ -17,8 +17,6 @@ private val log = KotlinLogging.logger {}
 
 /**
  * Implementation of the [CiagKpiService] with PEF (April '25 -> October '25) specific behaviours
- *
- * Enabled when the property `ciag-kpi-processing-rule` is set to `PEF` via [CiagKpiServiceFactory]
```

Comment:
Not technically true any more; I don't think we have that factory class anymore! 🤣